chapter_deep-learning-computation/custom-layer.md

```diff
@@ -8,6 +8,7 @@
 
 ```{.python .input}
 from mxnet import gluon, np, npx
+from mxnet.gluon.parameter import Parameter
```

Comment:
Thanks, but we need to keep the code consistency with our English version.

Reply:
ok